AK/Base64.h

```diff
@@ -35,16 +48,30 @@ consteval auto base64_lookup_table()
     return table;
 }
 
+consteval auto base64url_lookup_table()
```

Comment:
Isn't it possible to use a template to factorize this function too?

Reply:
Hmm probably, but it's so simple that I didn't think it was worth it. The base64_lookup_table() is used directly in LibWeb/Infra/Base64 for decode_forgiving_base64, so I didn't really want to mess with the consteval methods too much.